manager/testdata/new-blueprints/scenario4/plotter.yaml

```diff
@@ -56,6 +56,44 @@ spec:
         bucket: srcbucket
         object: inventory.parq
     dataformat: parquet
+  - assetId: "m4d-notebook-sample/paysim-step2" # the sink of step 2, consumed by step 1
+    cluster: thegreendragon
+    connection: 
+      arrow-flight:
+        endpoint:  app1-ns1-arrow-transform.m4d-blueprints
+        assetId: "m4d-notebook-sample/paysim" # always the same as the assetId known to the user (assetId or advertisedAssetId)
+  - assetId: "m4d-notebook-sample/paysim-step1" # used by workload to read the data
```

Comment:
I think having all the intermediate "virtual" assets in here is misleading. These are not assets that are specified by the user. And also in order to know what the steps are really doing one have to look up the assets in the top of the plotter. I specifically wanted to define everything within the step because it improves readability from the step point of view.